kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -43,537 +13,18 @@ class KafkaCheck(AgentCheck):
-
-        # not all brokers might return a good response... Try all of them
-        coord_id = None
-        for _ in range(self._broker_retries):
-            for broker in client.cluster.brokers():
-                try:
-                    coord_resp = self._make_blocking_req(client, request, node_id=broker.nodeId)
-                    # 0 means that there is no error
-                    if coord_resp and coord_resp.error_code == 0:
-                        client.cluster.add_group_coordinator(group, coord_resp)
-                        coord_id = client.cluster.coordinator_for_group(group)
-                        if coord_id is not None and coord_id >= 0:
-                            return coord_id
-                except AssertionError:
-                    continue
-            else:
-                coord_id = None
-
-        return coord_id
-
-    def _process_highwater_offsets(self, response):
-        highwater_offsets = {}
-        topic_partitions_without_a_leader = []
+    __NAMESPACE__ = 'kafka'
 
-        for tp in response.topics:
-            topic = tp[0]
-            partitions = tp[1]
-            for partition, error_code, offsets in partitions:
-                if error_code == KAFKA_NO_ERROR:
-                    highwater_offsets[(topic, partition)] = offsets[0]
-                    # Valid error codes:
-                    # https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-PossibleErrorCodes.2
-                elif error_code == KAFKA_UNKNOWN_ERROR:
-                    self.log.error(
-                        "Kafka broker returned UNKNOWN (error_code -1) for topic: %s, partition: %s. "
-                        "This should never happen.",
-                        topic,
-                        partition,
-                    )
-                elif error_code == KAFKA_UNKNOWN_TOPIC_OR_PARTITION:
-                    self.log.warn(
-                        "Kafka broker returned UNKNOWN_TOPIC_OR_PARTITION (error_code 3) for "
-                        "topic: %s, partition: %s. This should only happen if the topic is "
-                        "currently being deleted.",
-                        topic,
-                        partition,
-                    )
-                elif error_code == KAFKA_NOT_LEADER_FOR_PARTITION:
-                    self.log.warn(
-                        "Kafka broker returned NOT_LEADER_FOR_PARTITION (error_code 6) for "
-                        "topic: %s, partition: %s. This should only happen if the broker that "
-                        "was the partition leader when kafka_client.cluster last fetched metadata "
-                        "is no longer the leader.",
-                        topic,
-                        partition,
-                    )
-                    topic_partitions_without_a_leader.append((topic, partition))
+    def __init__(self, name, init_config, instances):
```

Comment:
You can remove this.

Reply:
Okay, I just thought I'd have it already there for the next PR